vizro-core/src/vizro/static/js/models/dashboard.js

```diff
@@ -1,3 +1,9 @@
 export function _update_dashboard_theme(on) {
   return on ? "vizro_dark" : "vizro_light";
 }
+
+export function _collapse_nav_panel(n_clicks, is_open) {
+  if (n_clicks) return !is_open;
+
+  return is_open;
+}
```

Comment:
Switching the collapse icon between `chevron_right` and `collapse_left` in this clientside callback makes sense to me as a possible solution (I think you mentioned it in standup?) but do you get the same nice animation that you get with [`dbc.Accordion`](https://dash-bootstrap-components.opensource.faculty.ai/docs/components/accordion/) when you click the arrows? I guess not.

Reply:
I think we should be able to achieve that via CSS transitions: https://developer.mozilla.org/en-US/docs/Web/CSS/CSS_transitions/Using_CSS_transitions

@nadijagraca, if you can check that out - that would be great!